cloudtrail-lambda-dynamo-cdk/src/lib/tag-compliance-stack.ts

```diff
@@ -11,24 +12,21 @@ export class TagComplianceStack extends cdk.Stack {
   constructor(scope: Construct, id: string, props?: cdk.StackProps) {
     super(scope, id, props);
 
-    const region = cdk.Aws.REGION;
-    const accountId = cdk.Aws.ACCOUNT_ID;
-
-    const table = new dynamodb.Table(this, 'resourceCreationTable', {
-      tableName: 'resource-creation-table',
+    const table = new dynamodb.Table(this, 's3ObjectsTable', {
+      tableName: 's3-objects-table',
       partitionKey: {
-        name: 'resource_arn', // name may be changed
+        name: 'object_arn', 
         type: dynamodb.AttributeType.STRING
       },
       stream: dynamodb.StreamViewType.NEW_IMAGE,
-      removalPolicy: cdk.RemovalPolicy.DESTROY // can be adjusted as deemed necessary
+      removalPolicy: cdk.RemovalPolicy.DESTROY 
     });
 
     // random generated number appended to bucket name, to create unique name
-    const generatedNum = Math.ceil(Math.random() * 10);
+    const generatedNum = Math.ceil(Math.random() * 100);
 
-    const bucket = new s3.Bucket(this, 'resourceCreationLogs', {
```

Comment:
Suggest also outputting the bucket name when the stack is deployed so its easier to find out what it is for testing